### PR TITLE
fix: Remove workaround of total token supply

### LINKED ISSUE #37: Remove workaround of total token supply
We used a temporary workaround because we could not wait for MDW fix

Once this https://github.com/aeternity/ae_mdw/issues/1244  will be merged and deployed we can revert workaround

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #37 

## Demo
MDW provides slightly different amount, but I guess it's more accurate than AEknow
image

## Checklist:
- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] Use fixed value from MDW

### CODE REVIEW

**Review by michele-franchi — CHANGES_REQUESTED** [review]
We cannot "guess" if the values is correct, it should be verified. If you look at the inflation curve the current value should be around "415017440". Could you please check and verify with mdw why there's a difference?

**Comment by janmichek** [comment]
> We cannot "guess" if the values is correct, it should be verified. If you look at the inflation curve the current value should be around "415017440". Could you please check and verify with mdw why there's a difference?

I mean, I would better rely on our solution then third party unknown formu. Question raised here https://github.com/aeternity/ae_mdw/pull/1252#issuecomment-1568098872

**Review by lukeromanowicz — COMMENTED** [review]
The code looks good to me but I'm not able to verify if the new value is correct

**Comment by janmichek** [comment]
@michele-franchi Something is being discussed .) https://github.com/aeternity/ae_mdw/pull/1252#issuecomment-1568922984

**Review by michele-franchi — CHANGES_REQUESTED** [review]
image

Distribution should also include the burned amount, could you please adjust it?

**Comment by janmichek** [comment]
> image
> 
> Distribution should also include the burned amount, could you please adjust it?

Good catch, ser. Fixed.

**Review by lukeromanowicz — APPROVED** [review]
Apart from one small issue, it looks good to me. I'm approving in advance

File: src/stores/marketStats.js [hunk 1]
```diff
@@ -14,18 +14,19 @@ export const useMarketStatsStore = defineStore('marketStats', () => {
...
-      ? circulatingSupply.value + Number(blockchainStatsStore.burnedCount)
+    blockchainStatsStore.totalTokenSupply && blockchainStatsStore.burnedCount
+      ? Number(blockchainStatsStore.totalTokenSupply) + Number(blockchainStatsStore.burnedCount)
       : null,
   )
 
-  const distributionPercentage = computed(() =>
-    distribution.value ? (distribution.value / MAX_AE_DISTRIBUTION * 100).toFixed(2) : null,
+  const distributionPercentage = computed(() => {
+    console.log('distribution.value', distribution.value)
```
[the remark this answers is not in the document]
Comment (janmichek) [reply src/stores/marketStats.js]: Ooops, thanks for second eyes. Removed

**Review by michele-franchi — APPROVED** [review]
Looks good 👏

**Review by CharisSiarampalis — APPROVED** [review]

**Pull request merged**